samples/client.c

```diff
@@ -1250,12 +1250,12 @@ int main(int argc, char *argv[])
     }
 
     /*
-     * Schedule an interrupt in 3 seconds from now in the old server and then
+     * Schedule an interrupt in 10 seconds from now in the old server and then
```

Comment:
separate PR please?

Does this fix the constant CI hangs/timeouts?

Reply:
Yes, OK will separate them.